web-client/example/module.js

```diff
@@ -46,7 +53,17 @@ init().then(async () => {
 
     document.querySelector('#address-book').addEventListener("click", async () => {
         let contacts = await client.getAddressBook();
-        console.table(contacts);
+        console.log(contacts
+            .sort((a, b) => {
+                // Sort seeds first
+                const aIsSeed = a.address.includes('seed');
+                const bIsSeed = b.address.includes('seed');
```

Comment:
Mmm, this might not hold true forever

Reply:
What might not hold true forever?

Reply:
That seeds always include `seed` in their addresses

Reply:
Ah true, and also other nodes might just include `seed` in their hostnames.

Reply:
It's just a sorting for better scannability in the demo page, nothing for production.